src/Core/Components/MenuButton/FluentMenuButton.razor

```diff
@@ -9,9 +9,16 @@
     </FluentButton>
     <FluentOverlay @bind-Visible="@_visible" Transparent="true" FullScreen="true" />
     <FluentMenu @ref="Menu" Anchor="@_buttonId" aria-labelledby="button" Style="@MenuStyleValue" @bind-Open=@_visible @onmenuchange=OnMenuChangeAsync>
-        @foreach (KeyValuePair<string, string> item in Items)
+        @if (Items is not null)
```

Comment:
Maybe interesting to remove this test and to allow both: `Items` and next `ChildContent`.
Just to add this information in the doc :-)

```razor
@foreach (KeyValuePair<string, string> item in Items)
{
    <FluentMenuItem Id="@item.Key">@item.Value</FluentMenuItem>
}

@ChildContent
```

Reply:
Yes, that is even better. Made that change and added documentation.